src/Impostor.Api/Net/Messages/C2S/Message08EndGameC2S.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Impostor.Api.Innersloth;
+
+namespace Impostor.Api.Net.Messages.C2S
+{
+    public class Message08EndGameC2S
+    {
+        public static void Serialize(IMessageWriter writer)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public static void Deserialize(IMessageReader reader, out GameOverReason gameOverReason, out bool showAd)
+        {
+            gameOverReason = (GameOverReason)reader.ReadByte();
+            showAd = reader.ReadBoolean(); // Perhaps we could disable ads on Impostor?
```

Comment:
I think it's in Impostor's best interest to leave that as it is. Otherwise we'd be stripping out a revenue income stream from Innersloth, and not sure what type of legal implications that would have.

Reply:
Good one, I agree. @reecebenson